### PR TITLE
Updates for moveit_config pkg

### PR DESCRIPTION
**Pull request by davetcoleman**
Added new planning groups: both_arms and whole_body
Added default starting pose 'home'
Applied latest changes from setup assistant (OMPL, ClearOctomapService)

Note: should we add an indigo-devel pkg now?

### CODE REVIEW

**Comment by shaun-edwards** [comment]
@davetcoleman, do we want to branch for indigo (or possible jade)?

**Comment by davetcoleman** [comment]
Branching for indigo makes sense to me

**Comment by gavanderhoorn** [comment]
Afaict, branching is required: the new moveit capabilities don't exist in the Hydro version.

File: hilgendorf_moveit_config/config/hilgendorf.srdf [hunk 1]
```diff
@@ -21,11 +21,36 @@
...
+        <joint name="right_wrist_1_joint" value="-3.1415" />
+        <joint name="right_wrist_2_joint" value="0" />
+        <joint name="right_wrist_3_joint" value="0" />
+    </group_state>
     <!--END EFFECTOR: Purpose: Represent information about an end effector.-->
     <end_effector name="right_gripper" parent_link="right_gripper_target" group="right_eef" />
     <end_effector name="left_gripper" parent_link="left_gripper_target" group="left_eef" />
     <!--VIRTUAL JOINT: Purpose: this element defines a virtual joint between a robot link and an external frame of reference (considered fixed with respect to the robot)-->
-    <virtual_joint name="fixed_base" type="fixed" parent_frame="world" child_link="base_link" />
+    <virtual_joint name="virtual_joint" type="fixed" parent_frame="world" child_link="base_link" />
```
Comment (gavanderhoorn) [inline hilgendorf_moveit_config/config/hilgendorf.srdf:53]: Just curious: why the change to `virtual_joint`?

File: hilgendorf_moveit_config/config/joint_limits.yaml [hunk 2]
```diff
@@ -1,61 +1,64 @@
...
-    max_acceleration: 5
\ No newline at end of file
+    max_velocity: 3.2
+    has_acceleration_limits: false
+    max_acceleration: 0
+  right_wrist_3_joint:
+    has_velocity_limits: true
+    max_velocity: 3.2
+    has_acceleration_limits: false
+    max_acceleration: 0
```
Comment (gavanderhoorn) [inline hilgendorf_moveit_config/config/joint_limits.yaml:64]: This update removes all acceleration limits. Do we care?

Comment (davetcoleman) [inline hilgendorf_moveit_config/config/joint_limits.yaml:64]: Ah I missed that over-write from the Setup Assistant, hadn't realized they were different. I can add those back

File: hilgendorf_moveit_config/config/hilgendorf.srdf (hunk 1 above)
Comment (davetcoleman) [inline hilgendorf_moveit_config/config/hilgendorf.srdf:53]: This is the standard that I've seen from original PR2 SRDF that I have been using... and unfortunately I have some dumb code in moveit_setup_assistant that does not work (fails gracefully) if the vjoint is not named this

**Comment by davetcoleman** [comment]
I have a new indigo branch that I'll PR to the new `indigo-devel` branch once someone makes it. I do not have commit access to add the branch.

File: hilgendorf_moveit_config/config/hilgendorf.srdf (hunk 1 above)
Comment (gavanderhoorn) [inline hilgendorf_moveit_config/config/hilgendorf.srdf:53]: Ok. So would you recommend we change the [ros-i tutorial](http://wiki.ros.org/Industrial/Tutorials/Create_a_MoveIt_Pkg_for_an_Industrial_Robot) that recommends to set this to `FixedBase` then?

Comment (gavanderhoorn) [inline hilgendorf_moveit_config/config/hilgendorf.srdf:53]: Again, just curious, but could you point me to that code in MSA? A quick [search](https://github.com/ros-planning/moveit_setup_assistant/search?utf8=%E2%9C%93&q=virtual_joint) doesn't show anything I'd call 'dumb' (or that hard codes the name to `virtual_joint`)?

Comment (davetcoleman) [inline hilgendorf_moveit_config/config/hilgendorf.srdf:53]: i'm sorry, i did not mean to say moveit_setup_assistant, but my own package moveit_visual_tools. the hack is [here](https://github.com/davetcoleman/moveit_visual_tools/blob/jade-devel/src/moveit_visual_tools.cpp#L1372). no, i don't recommend you change the tutorial's recommendation. i didn't know ROS-I had a standard and I didn't think anyone would care that I updated the hilgendorf's vjoint name since it seems like an inactive repo

**Comment by shaun-edwards** [comment]
Created the `indigo-devel` branch.
